contrib/src/main/resources/reference.conf

```diff
@@ -0,0 +1,10 @@
+akka {
+  stream {
+    contrib {
+      # Waiting time after inner flow provided by the user for unfold flow API cancels upstream,
+      # to get also the downstream cancelation (as graceful completion or failure which is propagated).
+      # If inner flow fails to complete/fail downstream, stage is failed with an IllegalStateException.
+      unfold-flow-timeout = 1m
+    }
+  }
+}
```

Comment:
Maybe add a header block with "Akka Streams Contrib" and a short description like in the regular akka reference.conf files?